.github/workflows/check-actions.yaml

```
@@ -0,0 +1,13 @@
+---
+jobs:
+  check-actions:
+    name: 'Check actions'
+    runs-on: 'ubuntu-latest'
+    steps:
+    - if: '${{ github.event_name != ''merge_group'' }}'
+      uses: 'econia-labs/common/.github/actions/check-actions@ECO-3009'
```

```suggestion
      uses: 'econia-labs/common/.github/actions/check-actions@main'
```

(once https://github.com/econia-labs/common/pull/27 merges)

---

Addressed in a593edcd